app/controllers/open_source_controller.rb

```diff
@@ -1,4 +1,6 @@
 class OpenSourceController < ApplicationController
+  layout 'sidebar_metrics'
```

Comment:
why do we have 2 different layouts?

Reply:
because of the tab in navbar (Tech Blog) which does not use the sidebar